<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Monster_Saving_Throws</class>
 <widget class="QDialog" name="Add_Monster_Saving_Throws">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monster Saving Throws</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 55, 0, 255);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Str&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Saving_Throws_Save_Saving_Throws_Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);</string>
   </property>
   <property name="text">
    <string>Save Saving Throws</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Int&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Saving_Throws_Con">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Saving_Throws_Wis">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Saving_Throws_Dex">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Con&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Saving_Throws_Int">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Saving_Throws_Cha">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cha&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wis&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Saving_Throws_Str">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Main_Menu_Label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Saving Throws&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dex&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Saving_Throws_Save_Saving_Throws_Button</sender>
   <signal>clicked()</signal>
   <receiver>Add_Monster_Saving_Throws</receiver>
   <slot>Add_Monster_Saving_Throws_Save_DB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>1469</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Add_Monster_Saving_Throws_Save_DB()</slot>
 </slots>
</ui>
